src/control/AudioController.h

```diff
@@ -19,13 +19,15 @@
 #include <util/audio/AudioPlayer.h>
 #include <gui/toolbarMenubar/ToolMenuHandler.h>
 
-class AudioController
+struct AudioController final
```

Comment:
Why exactly are you changing this to a struct?

Reply:
Because we write all member variables downside and constructors upside. Most constructors need to be public (in this case they need), so writing struct increases readability. Structs and classes are exactly the same except for the default visibility. So writing struct{...}; is the same as writing class{public:..}; struct is exactly what we want in all cases.

Reply:
I'm not really a fan of using structs here as it invites people to make attributes public by accident. Better would be to just reorder the classes, so that variables are top and everything else comes afterwards.